.github/workflows/build-and-test-orchard-core.yml

```diff
@@ -142,7 +142,7 @@ jobs:
         location: ${{ inputs.build-directory}}
 
     - name: Tests
-      uses: Lombiq/GitHub-Actions/.github/actions/test-dotnet@dev
+      uses: Lombiq/GitHub-Actions/.github/actions/test-dotnet@issue/OSOE-392
```

Comment:
Why is this necessary? There're no changes in `test-dotnet`.

Reply:
I have changed the connection string previously in `Invoke-SolutionTests`, but it was merged in OSOE-403, that's why the branch is changed, but yes, this is unnecessary now.